prow/cmd/hook/main.go

```diff
@@ -133,6 +134,8 @@ func main() {
 	if err != nil {
 		logger.WithError(err).Fatal("Error getting git client.")
 	}
+	// set github cerdential to clone repo.
+	gitClient.SetCredentials(*githubUsername, oauthSecret)
```

Comment:
why do you need user name?  shouldn't token suffice?

Reply:
user name used for ```git clone```, when use `git clone` will require user input username and token

Reply:
I guess my question is why upstream doesn't need username?  is it because of private repo?